### ISSUE TITLE
Better error messages

### ISSUE THREAD

**Issue by eventualbuddha** [esperantojs/esperanto#57]
We should do something like 6to5's error messages:

```
$ echo "import this from 'foo';" | ./bin/6to5/index.js
SyntaxError: stdin: Unexpected token (1:7)
> 1  | import this from 'foo';
     |       ^
  2  |
```

**Comment by Rich-Harris** [comment]
Yes, this would be helpful. Do you think it should be part of the error object itself, or that the CLI should put it together from file, line and column properties of the error? I lean towards the latter, though a) maybe that's just me, and b) I suppose they're not mutually exclusive

**Comment by eventualbuddha** [comment]
If we do it in the error message object then all the tools that use esperanto get it for "free", which is nice. I'm inclined to choose that option for that reason alone.
